python/unblob/handlers/filesystem/yaffs.py

```diff
@@ -8,8 +8,8 @@
 
 import attrs
 from structlog import get_logger
-from treelib import Tree
-from treelib.exceptions import NodeIDAbsentError
+from treelib import Tree  # type: ignore
```

Comment:
pyright says it would be okay to import from `treelib.tree`
```
   /home/runner/work/unblob/unblob/python/unblob/handlers/filesystem/yaffs.py:11:21 - error: "Tree" is not exported from module "treelib"
    Import from "treelib.tree" instead (reportPrivateImportUsage)
```

Reply:
fixed